editor/import/resource_importer_scene.cpp

```diff
@@ -302,159 +262,89 @@ Node *ResourceImporterScene::_fix_node(Node *p_node, Node *p_root, Map<Ref<Mesh>
 		}
 	}
 
-	String name = p_node->get_name();
+	const auto [node_name, node_hint] = _get_node_import_hint(p_node->get_name());
```

Comment:
Auto is banned.

Reply:
Sorry, I just returned `Pair` and used `auto` to decompose the variables. Replaced by normal use of `Pair`. Maybe create a structure for it? Using `first` and `second` looks bad.